.azure/pipelines/jobs/default-build.yml

```diff
@@ -83,6 +83,7 @@ jobs:
     enableTelemetry: true
     helixRepo: dotnet/aspnetcore
     helixType: build.product/
+    useHostedUbuntu: true
```

Comment:
Whether or not queue names are case-sensitive, this placement was the root cause. I didn't remember that this section is about passing parameters into the job.yml template when talking to @NTaylorMullen about this. This parameter default needs to be near the top of the file ☹️